### PR TITLE
FAQ page

### PR DESCRIPTION
**Pull request by ipopescu**
I added an FAQ page based on what I received from marketing. Please review and merge. Thanks!
Screenshot 2021-02-24 at 14 02 15
Screenshot 2021-02-24 at 14 02 31
Screenshot 2021-02-24 at 14 05 12

### CODE REVIEW

File: faq/index.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+When is Casper’s mainnet launch?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Casper’s mainnet launch will occur in Q1 2021.
+
+How does Casper solve the trilemma of security, decentralization, and scalability?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+As a Proof of Stake network, Casper relies on a set of validators to uphold the network. PoS validators do not require the same hardware setup as Proof of Work networks, allowing validators to be more geographically decentralized and eliminating the centralizing pressures of economies of scale. PoS also enables scalability mechanisms like sharding, allowing the network to process multiple transactions simultaneously, improving overall throughput.
```
Comment (zie1ony) [inline faq/index.rst:21]: > "like sharding"
We don't have sharding now. I would change it to sth like: "PoS will allow to implement scalability in future eg: sharding".

File: faq/index.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Why is Casper a good fit for enterprises and developers?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Casper is optimized for enterprise and developers through three main features: upgradeable contracts, developer-friendly languages, and predictable network fees.
+
+* **Upgradeable Contracts**. Casper enables on-chain smart contracts to be directly upgraded, removing the need for complex and migration processes and making it easier to patch smart contract vulnerabilities.
+
+* **Developer-Friendly Languages**. Casper supports developers building with WebAssembly. The network’s development ecosystem is designed to be familiar to existing Web2 developers instead of being written in a proprietary language like Solidity. This provides an easier path for developers and businesses to start building with Casper immediately.
+
+* **Predictable Network Fees**. Casper incentives active and diverse network behavior by establishing consistent, predictable, and transparent gas costs - eliminating volatility and improving both developer and user experience.
```
Comment (zie1ony) [inline faq/index.rst:35]: This feature is not implemented, yet.

File: faq/index.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Who has partnered with CasperLabs to build on the Casper network?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+CasperLabs has established a diverse ecosystem of participants who will be building, staking, and validating on Casper from day one, spanning blockchain, enterprise technology, and financial services. Among others, CasperLabs has established relationships with BitGo, Huobi, Ledger Leap, Chainlink, Metis, and Cardinal Cryptography. Companies including IPwe and Broadleaf are already using the Casper Network to unlock new value by tokenizing existing assets. 
+
+To learn more about how to work within the Casper ecosystem, contact neil@casperlabs.io.
+
+CSPR Token
+=============
+
+What is CSPR? What role does it play in the Casper network?
```
Comment (zie1ony) [inline faq/index.rst:69]: This line is one letter to long. It renders the hook link in another line on my end:

cspr

**Review by zie1ony — CHANGES_REQUESTED** [review]

File: faq/index.rst (hunk 1 above)
Comment (ipopescu) [reply faq/index.rst:21]: Changed to: "PoS also enables scalability mechanisms like sharding, a forthcoming feature to allow the network to process multiple transactions simultaneously, improving overall throughput."

File: faq/index.rst (hunk 3 above)
Comment (ipopescu) [reply faq/index.rst:69]: Great catch. I made it shorter: "What is CSPR, and its role in the Casper network?"

File: faq/index.rst (hunk 2 above)
Comment (ipopescu) [reply faq/index.rst:35]: I'll need to check with marketing, but I propose these changes: 
1. say that the predictable network fees are forthcoming: "three main features: upgradeable contracts, developer-friendly languages, and forthcoming predictable network fees."
2. Add "in development" and change the 3rd paragraph to say:
**Predictable Network Fees (In Development)**. Casper **intends to incentivize** active and diverse network behavior by establishing consistent, predictable, and transparent gas costs - eliminating volatility and improving both developer and user experience.

**Review by zie1ony — APPROVED** [review]

**Pull request merged**